numpy/core/_add_newdocs_scalars.py

```diff
@@ -218,10 +218,15 @@ def add_newdoc_for_scalar_type(obj, fixed_aliases, doc):
 
 add_newdoc_for_scalar_type('datetime64', [],
     """
-    A datetime stored as a 64-bit integer, counting from ``1970-01-01T00:00:00``.
```

Comment:
Doesn't it make sense to keep this description too? This change seems to remove information about `datetime64` and add information about `datetime64.__new__`. Ideally we'd have both!